packages/blockchain-wallet-v4-frontend/src/routes/extensionRoutes/HomeRoutes.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { NavLink, Route, Switch } from 'react-router-dom'
+import { IconHistory, IconListBullets, IconNft } from '@blockchain-com/icons'
+import { bindActionCreators } from 'redux'
+import styled from 'styled-components'
+
+import { Text } from 'blockchain-info-components'
+import { actions } from 'data'
+import BottomBarLayout from 'layouts/extension/BottomBarLayout'
+
+// FIXME:
+// eslint-disable-next-line import/no-named-as-default
+import CoinView from '../../scenes/extension/CoinView'
+
+const HomeNavbarItem = styled(NavLink)`
```

Comment:
Should the style components be in a separate file called `HomeRoutes.styles.ts`?

Reply:
Fine to leave this here